policyengine_us/variables/gov/states/sc/tax/income/credits/college_tuition/sc_tuition_credit.py

```diff
@@ -0,0 +1,33 @@
+from policyengine_us.model_api import *
+
+
+class sc_tuition_credit(Variable):
+    value_type = float
+    entity = Person
+    label = "South Carolina Tuition Credit"
+    defined_for = StateCode.SC
+    unit = USD
+    definition_period = YEAR
+    reference: "https://dor.sc.gov/forms-site/Forms/I319_2021.pdf#page=2"
+
+    def formula(person, period, parameters):
+        p = parameters(period).gov.states.sc.tax.income.credits.college_tuition
+
+        # line 1
+        total_hours = person("total_college_hours", period)
+        # line 2
+        qualified_tuition_expenses = person(
+            "qualified_tuition_expenses", period
+        )
+        # line 3
+        annual_hour_requirement = p.annual_hour_requirement
+        tuition_limit = (
+            p.max_amount.tuition * total_hours / annual_hour_requirement
```

Comment:
Don't need an intermediate variable

But more importantly, if this is a requirement then it's not a ratio but an eligibility flag. If that's right please make a `sc_tuition_credit_eligible` variable that compares hours to the requirement.
```suggestion
            p.max_amount.tuition * total_hours / p.annual_hour_requirement
```

Reply:
Actually the form looks to diverge from the law. Prorating the amount by hours would not have an effect if we also impose an eligibility constraint. Please remove the eligibility test and leave a comment indicating as much. If you can find legal justification for the form's logic even better.

Reply:
The law required at least 30 credit hours in a year 
![image](https://github.com/PolicyEngine/policyengine-us/assets/113824411/35c5fcdf-55af-45cd-a9b5-d9f97741e328)
but the tax form indicates that if a student only attends one semester and completes 15 credit hours, then the student is also eligible. 
![image](https://github.com/PolicyEngine/policyengine-us/assets/113824411/707f8f74-525b-4fb2-847d-979c2240f2ef)
So I think we do need an eligibility constraint. To follow the logic of the tax form, I can write the code 
`return where(sc_tuition_credit_eligible, min_(uncapped_credit, p.max_amount.credit), 0)` 
So far my code has not considered the one-semester case, do you want me to include that scenario as well?

Reply:
need another variable for semesters attended then